orbit/radius.py

```diff
@@ -472,10 +472,10 @@ def get_automated_feedback(self, attr):
             case _:
```

Comment:
I think this whole commit can just be squashed into the previous one since the second hunk that is adding the output into gradable_row is repeating the same change, and the first hunk is just rewriting the code from the previous commit.

Reply:
Ok nice that worked